### ISSUE TITLE
On accessing the FormItem inside Form, the associate label name is not announced to the screen reader user

### ISSUE THREAD

**Issue by C5362575** [SAP/ui5-webcomponents-react#5820]
### Bug Description

On accessing the FormItem, the associate label name is not announced to the screen reader Screen Reader: JAWS Version 2024.2403.3
A sample code is shared below for rederence
```
import {
  Form,
  FormItem,
} from "@ui5/webcomponents-react";

 <Form>
        <FormItem>
          label="labelText"
          aria-label="labelText"
        >
          <CustomComponent />
        </FormItem>
  </Form>
```

### Affected Component

Form,FormItem

### Expected Behaviour

All the contents and the information should be conveyed to the user, the corresponding label and group label should be read out, when each & every UI is accessed.

Relationships and information conveyed by the presentation (such as labels) should be identified by assistive technologies so user can connect the accessed UI with associated label.

 

### Steps to Reproduce

1.
2.
3.
...

### Priority

Medium

### UI5 Web Components Version

1.19.0

### Browser

Chrome

### Operating System

Windows 11 Enterprise, 23H2

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by ilhan007** [comment]
Hello @C5362575 the form in question is part of the UI5 Web Components For React and that's I am forwarding the issue to the  UI5 Web Components For React repository.

@MarcusNotheis @Lukas742 

BR,
ilhan

**Comment by C5362575** [comment]
Hi @MarcusNotheis , @Lukas742  

Please look into this reported issue on an urgent basis. we need to close these accessibility bugs

**Comment by C5362575** [comment]
Hi @MarcusNotheis , @Lukas742  can you please let us know if the team is looking into this issue

**Comment by Lukas742** [comment]
Hi @C5362575 

The linked PR will enhance the announcements for standard UI5 Web Components inputs. Please note that for custom React components rendering a UI5 Web Component input, it is mandatory to pass the `accessibleNameRef` prop to the input Web Component. Without this, the association cannot be made.
